pkg/loglevel/loglevel.go

```diff
@@ -61,7 +61,7 @@ func UpdatePodSpec(podSpec *corev1.PodSpec, cntName string, level operatorv1.Log
 		return fmt.Errorf("cannot modify the arguments for container %s", cnt.Name)
 	}
 	flags.SetOption("--v", kLog.String())
-	klog.InfoS("container klog level", "container", cnt.Name, "-v", kLog.String())
+	klog.InfoS("container klog level", "container", cnt.Name, "--v", kLog.String())
```

Comment:
I prefer the other way around: single-letter identifiers should use a single dash
```
-v
--long-option
```

Reply:
addressing this here: https://github.com/openshift-kni/numaresources-operator/pull/848